crates/ide/src/completion/generated_lint_completions.rs

```diff
@@ -0,0 +1,5 @@
+//! Generated file, do not edit by hand, see `xtask/src/codegen`
```

Comment:
it might be nice to run rustfmt on that?

Reply:
I added the function to do that but it seems that rustfmt doesn't want to format this file. Any idea ?